libs/sdk-backend-spi/src/workspace/dashboards/index.ts

```diff
@@ -43,6 +43,14 @@ export interface IGetDashboardOptions {
  * @alpha
  */
 export interface IGetScheduledMailOptions {
+    /**
+     * Specify if information about the users that created/modified the scheduled email should be loaded.
+     * Defaults to false.
```

Comment:
Please move "Defaults to false" + the other sentence below `@remarks` annotation. (see `docs/documentation.md`)